tests/Request/CreateAdjustmentRequestTest.php

```diff
@@ -8,6 +8,7 @@
 
 class CreateAdjustmentRequestTest extends TestCase
 {
+    protected $program = 'sharecare';
```

Comment:
let's default all of these to 'alldigitalrewards' -- i've updated other examples in this SDK to match prior and in RewardStacks dev seeder to reflect the same thing,